aws/tf/modules/sra/databricks_workspace/workspace_security_modules/cluster_configuration/cluster_configuration.tf

```diff
@@ -31,10 +31,11 @@ resource "databricks_cluster_policy" "example" {
 
 // Cluster Creation
 resource "databricks_cluster" "example" {
-  cluster_name  = "Shared Cluster"
-  spark_version = data.databricks_spark_version.latest_lts.id
-  node_type_id  = "i3.xlarge"
-  policy_id     = databricks_cluster_policy.example.id
+  cluster_name       = "Shared Cluster"
+  data_security_mode = "USER_ISOLATION"
+  spark_version      = data.databricks_spark_version.latest_lts.id
+  node_type_id       = "i3.xlarge"
```

Comment:
Should this be the Nitro instances if compliance_security_profile is enabled? I3en

Reply:
Good call out, I'll add this as a flag with ESC